lib/twine/formatters/abstract.rb

```diff
@@ -3,17 +3,28 @@
 module Twine
   module Formatters
     class Abstract
-      attr_reader :strings
-      attr_reader :options
+      attr_accessor :strings
+      attr_accessor :options
 
-      def self.can_handle_directory?(path)
-        return false
-      end
-
-      def initialize(strings, options)
+      def initialize(strings = StringsFile.new, options = {})
```

Comment:
This changed with this PR. Since formatter instances are registered in `Twine::Formatters.formatters` formatters are usually created using the default arguments (see last line in `android.rb` for example). `Runner::formatter_for_format` sets the `strings` and `options` property before using a formatter. However it's true that only the unit tests use the parameters. So we could get rid of the parameters altogether.